script.js

```diff
@@ -1,12 +1,125 @@
-//You can edit ALL of the code here
+const rootElem = document.getElementById("root");
+  //Create search feature;
+rootElem.innerHTML = `
+  <div id="search-episodes">
+  <button class="home" type="button">HOME</button>
+  <span class="search-bar">Series</span>
+  <select id="series-list"></option></select>
+  <span class="search-bar">Episodes</span>
+  <select id="episode-list"></option></select>
+  <input type="search" class="search-episodes"  placeholder="Search keywords"><p class="counter"></p>
+  </div>
+  <div class="episodeContainer"></div>`; 
+
+//Onload function / API fetch / default Serie: 1
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+  fetch("https://api.tvmaze.com/shows/1/episodes")
```

Comment:
It's better if you keep URLs like this in a `const` since you don't expect them to change 

e.g.
```
const DefaultShow =  "https://api.tvmaze.com/shows/1/episodes"
fetch(DefaultShow)
```

This is because in very large programs it can get confusing what the URL means an by setting it as a const it means you remember.

Reply:
Done.